spack/psc/packages/psc/package.py

```diff
@@ -42,5 +42,7 @@ def cmake_args(self):
             'ON' if '+nvtx' in self.spec else 'OFF')]
         args += ['-DPSC_USE_RMM={}'.format(
             'ON' if '+rmm' in self.spec else 'OFF')]
+        args += ['-DBUILD_TESTING={}'.format(
+            'ON' if '+tests' in self.spec else 'OFF')]
```

Comment:
This is good, but it also needs a corresponding `variant` to actually work.

Reply:
Oh ya, I keep losing changes I've made bouncing around branches.  I forgot to add that when I put it in this pr branch